Controller/DynamicController.php

```diff
@@ -100,7 +108,7 @@ public function cgetAction(Request $request): Response
             $offset
         );
 
-        $entries = $this->dynamicListFactory->build($entries, $view);
+        $entries = $this->dynamicListFactory->build($entries, $locale, $view);
```

Comment:
Interestingly, this is actually fixing a bug. The second argument for the `build` method in `DynamicListFactoryInterface` is the locale. But until now, the `view` was passed instead.

```php
public function build(array $dynamics, string $locale, string $builder = 'default'): array;
```

The only reason it was working is because the third argument has a fallback and in the existing `DynamicListBuilder` the `locale` (which would be `default`) is not being used.